Homework1/Sort/Sort/BubbleSort.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+
+namespace Sort
+{
+    public class BubbleSort
+    {
+        public void BubbleSortArray(int[] arrayOfNumbers)
+        {
+            for (int i = 0; i < arrayOfNumbers.Length - 1; i++)
+            {
+                for (int j = arrayOfNumbers.Length - 1; j > i; j--)
+                {
+                    if (arrayOfNumbers[j - 1] > arrayOfNumbers[j])
+                    {
+                        int temporary = arrayOfNumbers[j];
+                        arrayOfNumbers[j] = arrayOfNumbers[j - 1];
+                        arrayOfNumbers[j - 1] = temporary;
+                    }
+                }
+            }
+        }
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Введите количество элементов в массиве");
+            var inputString = Console.ReadLine();
+            int numberOFElementsInArray = int.Parse(inputString);
+            int[] arrayOfNumbers = new int[numberOFElementsInArray];
+            for(int i = 0; i < arrayOfNumbers.Length; i++)
+            {
+                Random random = new Random();
```

Comment:
```suggestion
                var random = new Random();
```
В C# почти никогда не надо писать тип дважды на одной строке

Reply:
И создавать новый объект Random на каждой итерации цикла — очень плохая идея, он заново инициализируется  (так что по идее почти все числа в массиве должны быть равны), и сборщик мусора за создание сотни ненужных объектов спасибо не скажет. Random вообще должен быть один на всю программу.